showOutput3.js

```diff
@@ -985,6 +881,7 @@ document.getElementById('submit').addEventListener(
 
             keyObject = new Array(
                 '"label": "' + keyLabel +' (' + keyPins + ')"',
+                '"matrix": [' + obj.keyboard.keys[key].row + ", " + obj.keyboard.keys[key].col + "]",
```

Comment:
This change requires changing the indexes for `keyObject.splice()` on Lines 891-897 to `5` and `4`, respectively.

https://github.com/fauxpark/kbf_qmk_converter/blob/data-driven/showOutput3.js#L891-L897